playbooks/app_dev/APM_guidance.adoc

```diff
@@ -0,0 +1,156 @@
+
+Note that in this case very little additional work is required in order to enable APM and that no additional image configuration is needed.
+
+=== Market players
+
+
+Major market players in APM are:
+
+ * https://www.dynatrace.com/[Dynatrace]
+ * https://www.appdynamics.com/[appDynamics]
+ * https://newrelic.com/[New Relic]
+
+For more information about the APM tools landscape see Gartner’s “Magic Quadrant” report on APM tools (document code https://www.gartner.com/document/3551918?ref=exploremq[G00298377]), available through the Gartner site. 
+
+It is important to be aware that there is a trend for pure infrastructure monitoring tools to invade the APM space and for APM tools to invade the infrastructure space. In fact a new generation of tools try to do both. Some examples are:
+
+ * https://sysdig.com/[Sysdig]
+ * https://www.instana.com/[Instana]
+
+Below we have a series of link to help getting started with each of the mentioned products.
+
+==== Installing Dynatrace on OpenShift
+
+https://www.dynatrace.com/technologies/cloud-and-microservices/openshift-monitoring/[https://www.dynatrace.com/technologies/cloud-and-microservices/openshift-monitoring/] +
+https://www.dynatrace.com/partners/technology-partners/redhat/[https://www.dynatrace.com/partners/technology-partners/redhat/] +
+https://blog.openshift.com/author/dynatrace/[https://blog.openshift.com/author/dynatrace/]
+
+==== Installing AppDynamics on OpenShift
+
+https://blog.appdynamics.com/product/using-appdynamics-with-red-hat-openshift-v3/[https://blog.appdynamics.com/product/using-appdynamics-with-red-hat-openshift-v3/] +
+https://www.appdynamics.com/solutions/cloud-monitoring/openshift/[https://www.appdynamics.com/solutions/cloud-monitoring/openshift/] +
+https://blog.openshift.com/appdynamics-integration-with-openshift/[https://blog.openshift.com/appdynamics-integration-with-openshift/]
+
+==== Installing New Relic on OpenShift
+
+https://blog.openshift.com/using-new-relic-to-monitor-applications-on-openshift/[https://blog.openshift.com/using-new-relic-to-monitor-applications-on-openshift/]
+
+==== Installing Sysdig on OpenShift
+
+https://sysdig.com/openshift-monitoring/[https://sysdig.com/openshift-monitoring/] +
+https://support.sysdig.com/hc/en-us/articles/211421063-Sysdig-Install-OpenShift[https://support.sysdig.com/hc/en-us/articles/211421063-Sysdig-Install-OpenShift] +
+https://blog.openshift.com/openshift-ecosystem-using-sysdig-monitor-openshift/[https://blog.openshift.com/openshift-ecosystem-using-sysdig-monitor-openshift/]
+
+==== Installing Instana on OpenShift
+
+https://blog.openshift.com/deploying-instana-apm-natively-openshift-environment/[https://blog.openshift.com/deploying-instana-apm-natively-openshift-environment/] +
+https://instana.atlassian.net/wiki/spaces/DOCS/pages/15630376/Docker+scheduled#Docker(scheduled)-OpenShift[https://instana.atlassian.net/wiki/spaces/DOCS/pages/15630376/Docker+scheduled#Docker(scheduled)-OpenShift]
+
+== Distributed Tracing
```

Comment:
You might also want to mention tools such as Zipkin and NetFlix OSS.  The Red Hat Developers MSA [1] also has a demo of this with OpenTracing, Zipkin, and Jaeger.

[1] https://github.com/redhat-helloworld-msa/helloworld-msa

Reply:
I added the reference to hello world msa, I don't think that zipkin should be mentioned as it is obsoleted by Jaeger.